### ISSUE TITLE
Q: Manual Install?

### ISSUE THREAD

**Issue by ettore-syt** [catppuccin/tmux#123]
Just out of curiosity,

to install manually and not use TPM what is the step-by-step process? 

I trying ape-ing at it, but I guess grug brain not gud enuf

**Comment by sgoudham** [comment]
Hey :wave:, I'm not a tmux user but I see that the README contains a [Manual](https://github.com/catppuccin/tmux#manual) section, have you followed those instructions?

**Comment by ettore-syt** [comment]
Those instructions are quite unclear for the unfamiliar user, this question hopes to address that issue :)

**Comment by irohn** [comment]
> Author

+1 I would love an explanation as well

**Comment by irohn** [comment]
As a workaround I changed the .tmux output to echo instead of tmux and after adjusting it to .conf file format (with LLM help) I got the rough config:

```
# Status line
set-option -gq status on
set-option -gq status-bg "#1e1e2e"
set-option -gq status-justify left
set-option -gq status-left-length 100
set-option -gq status-right-length 100

# Message style
set-option -gq message-style "fg=#89dceb,bg=#313244,align=centre"
set-option -gq message-command-style "fg=#89dceb,bg=#313244,align=centre"

# Window and pane options
set-window-option -gq pane-border-status off
set-window-option -gq pane-active-border-style "fg=#fab387"
set-window-option -gq pane-border-style "fg=#313244"
set-window-option -gq pane-border-format ""

# Window status styles
set-window-option -gq window-status-activity-style "fg=#cdd6f4,bg=#1e1e2e,none"
set-window-option -gq window-status-separator ""
set-window-option -gq window-status-style "fg=#cdd6f4,bg=#1e1e2e,none"
set-window-option -gq window-status-format "[fg=#89b4fa,bg=#1e1e2e]█#[fg=#313244,bg=#89b4fa]#I#[fg=#89b4fa,bg=#313244,nobold,nounderscore,noitalics]█ #[fg=#cdd6f4,bg=#313244]#{b:pane_current_path}#[fg=#313244,bg=#1e1e2e,nobold,nounderscore,noitalics]█"

set-window-option -gq window-status-current-format "[fg=#fab387,bg=#1e1e2e]█#[fg=#1e1e2e,bg=#fab387]#I#[fg=#fab387,bg=#1e1e2e,nobold,nounderscore,noitalics]█ #[fg=#cdd6f4,bg=#1e1e2e]#{b:pane_current_path}#[fg=#1e1e2e,bg=#1e1e2e,nobold,nounderscore,noitalics]█"

# Status line content
set-option -gq status-left ""
set-option -gq status-right "[fg=#f5c2e7,bg=#1e1e2e,nobold,nounderscore,noitalics]#[fg=#1e1e2e,bg=#f5c2e7,nobold,nounderscore,noitalics] #[fg=#cdd6f4,bg=#313244] #W#[fg=#313244,bg=#313244,nobold,nounderscore,noitalics]█#[fg=#{?client_prefix,#f38ba8,#a6e3a1},bg=#313244,nobold,nounderscore,noitalics]#[fg=#1e1e2e,bg=#{?client_prefix,#f38ba8,#a6e3a1},nobold,nounderscore,noitalics] #[fg=#cdd6f4,bg=#313244] #S#[fg=#313244,bg=#313244,nobold,nounderscore,noitalics]█"

# Clock mode color
set-window-option -gq clock-mode-colour "#89b4fa"

# Mode style
set-window-option -gq mode-style "fg=#f5c2e7 bg=#585b70 bold"
```

**Comment by j-o-sh** [comment]
I just went through that process and it would be really cool if that could be part of the README as well. But the `run` command is your friend here ;)

1. Clone the `catppuccin/tmux` repo somewhere you like (I cloned it into `~/.config/tmux/plugins/catppuccin`
2. Add this line to your `tmux.conf`: `run ~/.config/tmux/plugins/catppuccin/catppuccin.tmux`
3. Set a `@catppuccin_flavour` or any of the config examples in the readme directly in your `tmux.conf`

That worked for me :)
